services/common/src/main/java/org/collectionspace/services/common/storage/jpa/JpaStorageUtils.java

```diff
@@ -335,6 +335,12 @@ public static AccountPermission getAccountPermissions(String csid, String curren
 
 	//FIXME: REM - This method should probably be moved to the AccountPermissionDocumentHandler
 	public static AccountRole getAccountRoles(String csid)
+		throws UnauthorizedException, DocumentNotFoundException {
+		return getAccountRoles(csid, null, null);
+	}
+
+	//FIXME: REM - This method should probably be moved to the AccountPermissionDocumentHandler
+	public static AccountRole getAccountRoles(String csid, String currentResource, String permissionResource)
```

Comment:
`String currentResource, String permissionResource` were added to this method signature, but they are never used. Any idea why?

Reply:
@ray-lee I have no idea why this change happened.  If I'm interpreting git's 'blame' correctly, it may be a change you made several years ago?  See https://github.com/cspace-deployment/services/commit/28390c0771ad1bbe743ed743d98f0d344c846845

Either way, it doesn't seem necessary so we should stick with the current LYRASIS 'master' branch version.